pom.xml

```diff
@@ -46,9 +46,9 @@
         <jotm-core.version>2.1.10-kuali-1</jotm-core.version>
         <jsp-api.version>2.3.3</jsp-api.version>
         <jsr311-api.version>1.1.1</jsr311-api.version>
-        <junit.version>5.10.1</junit.version>
+        <junit.version>5.10.2</junit.version>
         <ojdbc.version>19.18.0.0</ojdbc.version>
-        <opencsv.version>5.8</opencsv.version>
+        <opencsv.version>5.9</opencsv.version>
         <plexus-utils.version>3.0.24</plexus-utils.version>
         <mockito.version>4.9.0</mockito.version>
```

Comment:
Since we removed the PowerMock dependency from the POM a few weeks ago, can we now upgrade Mockito to the same version that KualiCo uses?